lib/oli/delivery/metrics.ex

```diff
@@ -1243,19 +1244,20 @@ defmodule Oli.Delivery.Metrics do
              "? / NULLIF(?, 0)",
              summary.num_first_attempts_correct,
              summary.num_first_attempts
-           )}
+           ), summary.num_first_attempts}
       )
 
     Repo.all(query)
-    |> Enum.into(%{}, fn {page_id, proficiency} ->
-      {page_id, proficiency_range(proficiency)}
+    |> Enum.into(%{}, fn {page_id, proficiency, num_first_attempts} ->
+      {page_id, proficiency_range(proficiency, num_first_attempts)}
     end)
   end
 
-  def proficiency_range(nil), do: "Not enough data"
-  def proficiency_range(proficiency) when proficiency <= 0.5, do: "Low"
-  def proficiency_range(proficiency) when proficiency <= 0.8, do: "Medium"
-  def proficiency_range(_proficiency), do: "High"
+  def proficiency_range(_, num_first_attempts) when num_first_attempts < 3, do: "Not enough data"
```

Comment:
This is the core of the logic to determine if `num_first_attempts` is less than 3, then simply return "Not enough data"